src/scanoss/agent/snippet_scan.h

```diff
@@ -0,0 +1,49 @@
+/**************************************************************
+ Copyright (C) 2010-2011 Hewlett-Packard Development Company, L.P.
```

Comment:
Please update the copyright information.

Reply:
Solved

Reply:
I can still see `Copyright (C) 2010-2011 Hewlett-Packard Development Company, L.P.`

Please check @scanoss-qg